index.d.ts

```diff
@@ -0,0 +1,36 @@
+interface Options {
+	/**
+	 * Number of concurrently pending promises returned by `mapper`.
+	 *
+	 * @default Infinity
+	 */
+	concurrency: number;
```

Comment:
Shouldn't `concurrency` be optional? It works without now, but if we add more options, it won't.

Reply:
Good point!